### ISSUE TITLE
After installing the patch to run the game and then closing the game, it can no longer be launched.

### ISSUE THREAD

**Issue by DemoJameson** [RPCS3/rpcs3#4494]
Everything is fine before installing the game patch, but it can only be launched once after installation. When I launch the game again, I get the following error message:

    S LDR: Updates found at /dev_hdd0/game/BCAS20071/!
    F LDR: Disc directory not found. Try to run the game from the actual game disc directory.
    E LDR: Failed to mount disc directory for the disc game 
    S LDR: Boot successful.

Seems to be a problem with `dev_hdd0\game\BCAS20071\PARAM.SFO`.
The original and corrupted version of the file [PARAM.SFO.zip](https://github.com/RPCS3/rpcs3/files/1957862/PARAM.SFO.zip)

RPCS3 Version: 0.0.5-6692
System: Win10
CPU: 8700K
GPU: 1080
Memory: 16G
log: [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/1956797/RPCS3.log.gz)

**Comment by Zangetsu38** [comment]
Disgea 3 have same issue

**Comment by AniLeo** [comment]
Removed game update link for obvious reasons

**Comment by dio-gh** [comment]
I'll be frank, not having the asian release of the game, I went and hacked up my EU copy to act like an asian one. I could not reproduce this issue at all. Naturally, my original param.sfo wasn't quite the same as the one in the ird, but it did boot properly and repeatably, and continued to do so after applying the asian 1.04 update as well.

What I noticed, is that OP's `param.sfo`s are off as well. The one he claims to be the "original" param.sfo is actually the one that's contained in the update, while the resulting "broken" param.sfo completely misses the product code and the game title.

The curious bit comes right here. I extracted the update pkg with nicba's [pkg extractor tool](https://github.com/Nicba1010/PS-Tools/releases/), that's how I found the aforementioned update sfo & "original" sfo match. However I noticed, that my own installed update matches neither of these. It's the same as OP's "broken" one, but with the product code and the game title actually filled in. Some other binary fields are different as well.

TL;DR: the param.sfo in the update vs. the param.sfo after installing the update in RPCS3 doesn't match, and I have no idea why. The issue I cannot reproduce however, as the resulting param.sfo is valid, and thus the game does boot properly.

[working_param_sfo.zip](https://github.com/RPCS3/rpcs3/files/3721324/working_param_sfo.zip)

Addendum: the param.sfo inside the update file states game version 1.03, but the post-install param.sfo states 1.04. How does this work, I have no idea.

Needs information, otherwise closeable, unless someone can actually reproduce. Maybe if @Zangetsu38 cares to tell us what version of Disgae 3 he actually meant, we'll figure it out; hopefully that won't be an asian release..

**Comment by Megamouse** [comment]
@DemoJameson please try this again with a clean install and report back.
If there is no response in a timely fashion we'll assume this is fixed, since package installation and PARAM.SFO creation have been improved over the past few years.

**Comment by Megamouse** [comment]
closing due to inactivity
